### PR TITLE
Centos7

### PR DESCRIPTION
**Pull request by jburel**
* Remove installation instructions for centOS 7

### CODE REVIEW

**Review by sbesson — CHANGES_REQUESTED** [review]
Dropping these installation pages completely makes sense. https://omero.readthedocs.io/en/stable/sysadmins/version-requirements.html#linux-centos-and-rhel should probably also be updated to mark CentOS/RHEL 7 as officially unsupported?

**Comment by jburel** [comment]
Related PRs https://github.com/ome/omero-install/pull/288 and https://github.com/ome/omeroweb-install/pull/96

**Review by sbesson — COMMENTED** [review]
Thanks, the only other place that has a clear reference to CentOS 7 as a target system that I could find is https://omero--2436.org.readthedocs.build/en/2436/sysadmins/python3-migration.html#choosing-a-platform.

I can see two routes here:
- either treat this page as historical and let it reflect the state of systems as of the Python 3 migration 4+ years ago
- or do a minimal update to the first paragraph pointing to the version requirements page for details on which operating system/Python version to migrate to

**Comment by jburel** [comment]
Thanks for pointing that out. 
If we treat as historical, we minimally need a warning
I will read it and see if adjusting makes more sense

File: omero/sysadmins/python3-migration.rst [hunk 1]
```diff
@@ -20,20 +20,14 @@ Basic steps
...
-The two recommended platforms, CentOS 7 and Ubuntu 18.04, have Python 3.6 as
-default installation and have therefore received the most testing which is why
-Python 3.6 is the preferred version of Python.
+The two recommended platforms, Rocky Linux/RHEL 9 and Ubuntu 22.04,
+have Python 3.9 and 3.10 respectively as
+default installation.
 
-Both Python 3.5 and 3.7 should work and are slated to have support added, but
-Python 3.6 has been the focus of testing during the migration.
+All Python 3.6+ versions should work.
```
Comment (sbesson) [inline omero/sysadmins/python3-migration.rst]: With recent OMERO.py versions Python 3.8 should be the current minimal requirement (soon to be bumped to Python 3.9 as 3.8 is becoming EOL in the Fall)

File: omero/sysadmins/python3-migration.rst [hunk 2]
```diff
@@ -20,20 +20,14 @@ Basic steps
...
 
-Similarly, other operating systems are slated for having support added, but help
-from the community would be very welcome! Obvious next candidates are CentOS 8
-and Ubuntu 20.04.
 
-Debian 9 is still on Python 3.5 and Debian 10 has moved to Ice 3.7.
-We have nonetheless an installation guide for Debian 9 with Python 3.5 and Ice 3.6
-but an installation guide for Debian 10 with Python 3.7 and Ice 3.6.
+Debian 10 has moved to Ice 3.7 but an installation guide for Debian 10 with Python 3.7 and Ice 3.6.
```
Comment (sbesson) [inline omero/sysadmins/python3-migration.rst]: This sentence is lacking a verb

**Review by sbesson — CHANGES_REQUESTED** [review]

File: omero/sysadmins/python3-migration.rst (hunk 2 above)
Comment (jburel) [reply omero/sysadmins/python3-migration.rst]: I will remove Debian 10 in a follow-up PR since it has also reached EOL (30/06)

**Review by sbesson — APPROVED** [review]

**Pull request merged**